common/proto/common/SDMS_Auth.proto

```diff
@@ -21,6 +21,11 @@ enum Protocol
     ID = 2;
 }
 
+enum AccessTokenType {
+    GENERIC = 1;
+    GLOBUS = 2;
+}
```

Comment:
@t-ramz  @AronPerez @nedvedba I thought we talked about expanding on this in Globus with different types? i.e. GLOBUS_AUTH vs something else?